packages/common/container-definitions/package.json

```diff
@@ -52,12 +55,25 @@
 		"@fluidframework/eslint-config-fluid": "^3.1.0",
 		"@microsoft/api-extractor": "^7.37.0",
 		"@types/events": "^3.0.0",
+		"copyfiles": "^2.4.1",
 		"eslint": "~8.50.0",
 		"eslint-plugin-deprecation": "~2.0.0",
 		"prettier": "~3.0.3",
 		"rimraf": "^4.4.0",
 		"typescript": "~5.1.6"
 	},
+	"fluidBuild": {
```

Comment:
I need to make a point to better understand how fluid-build works - is there a reason we can't capture this in our root configuration? Or will that only be possible once we've migrated everything?

Reply:
That's only possible once we've fully migrated. The root config describes defaults, but in this case we need to tell
fluid-build that build:docs is no longer a "script" task -- meaning fluid-build won't try to parse or invoke the script in
package.json. Instead, it uses this task definition. Basically this config makes the build:docs a "virtual task" for
this package only.